clients/src/test/java/org/apache/kafka/common/utils/ShellTest.java

```diff
@@ -60,15 +59,12 @@ public void testAttemptToRunNonExistentProgram() {
     }
 
     @Test
-    public void testRunProgramWithErrorReturn() throws Exception {
+    public void testRunProgramWithErrorReturn() {
         assumeTrue(!OperatingSystem.IS_WINDOWS);
```

Comment:
How about using conditional test (```DisabledOnOs```)?

Reply:
I think this is a good change we can apply across the board after we convert to JUnit 5.

Reply:
In order to keeping patch small, let me file separate ticket for this one.

see https://issues.apache.org/jira/browse/KAFKA-12189